CHANGES.rst

```diff
@@ -4,6 +4,11 @@ Change Log
 4.0 (unreleased)
 ----------------
 
+- Add property to clear session data at login boundary to the session auth
```

Comment:
```suggestion
- Add `clear_session_on_login` property to clear session data at login boundary to the session auth
```

Reply:
The actual name of the property is not relevant and will only confuse people because it never shows anywhere except in code. The property is a simple PropertyManager property and you toggle it in the ZMI.

Reply:
@dataflake Oh yeah, makes sense.